src/server/worker/common/common.go

```diff
@@ -43,14 +43,14 @@ func DatumID(inputs []*Input) string {
 	hash := pfs.NewHash()
 	for _, input := range inputs {
 		hash.Write([]byte(input.Name))
-		binary.Write(hash, binary.BigEndian, int64(len(input.Name)))
+		binary.Write(hash, binary.BigEndian, int64(len(input.Name))) //nolint:errcheck
```

Comment:
Yeah, this can't actually return an error.  binary.Write returns the underlying error from hash.Write, and hash.Write can't error.